.changeset/eleven-monkeys-return.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fix type declarations for custom message arguments
```

Comment:
[suggest] Thanks for adding the entry. Please fit the prefix to the existing format in `CHANGELOG.md` as below:

```suggestion
Fixed: type declarations for custom message arguments
```